Engine/Entities/Ants/Ant.cs

```diff
@@ -64,17 +66,17 @@ public Ant(string name, Transform transform, World world) : this(name, transform
         /// </summary>
         public int PerceptionMapPrecision { get; } = 24;
 
-        public TimeSpan PheromoneTimeSpan { get; protected set; } = TimeSpan.FromSeconds(500);
+        public int PheromoneTimeSpan { get; protected set; } = 60000;
 
         /// <summary>
         /// Delay between each emission of a pheromone.
         /// </summary>
-        public float PheromoneEmissionDelay { get; protected set; } = 0.3F;
+        public int PheromoneEmissionDelay { get; protected set; } = 5;
 
         /// <summary>
-        /// The timestamp of when the ant emitted a pheromone
+        /// The number of tick since the ant emitted a pheromone
         /// </summary>
-        public DateTime LastEmitTime { get; set; }
+        public int LastEmitTime;
```

Comment:
No default value?

Reply:
Nope, this is only the number of ticks since the ant emitted a pheromone aka a value incremented at each tick, starting from 0